### ISSUE TITLE
PHP 8.2 support

### ISSUE THREAD

**Issue by yhabteab** [Icinga/icingaweb2-module-businessprocess#403]
A collection of PHP 8.2 errors.

**Comment by yhabteab** [comment]
```bash
NOTICE: PHP message: PHP Deprecated:  Creation of dynamic property Icinga\Module\Businessprocess\Storage\LegacyConfigRenderer::$config is deprecated in /usr/share/icingaweb2-modules/businessprocess/library/Businessprocess/Storage/LegacyConfigRenderer.php on line 21
```

**Comment by nilmerg** [comment]
Resolved by #378
